index.rst

```diff
@@ -121,8 +128,9 @@ files take precedence.
 Supported Properties
 ====================
 
-EditorConfig file sections contain properties, which are name-value pairs
-separated by an equal sign (``=``). EditorConfig plugins will ignore
+EditorConfig file sections contain properties, which are key-value pairs
```

Comment:
"properties, which are key-value pairs ...." --> "declarations"?

Reply:
I'd prefer easier language here. Not everybody managing editorconfig-files may be a developer?

Reply:
Do you have a suggestion?

Reply:
Key-Value is fine for me - the Wikipedia-css page talks about "property"/"value"-pairs. I just would probably avoid "declaration" even though its technically the precise term it is probably needlessly complicated.

Reply:
I think it is important to use more precise language in a spec. How about employing some tautology?

> ... declarations, i.e., key-value pairs ...